.gitmodules

```diff
@@ -34,10 +34,6 @@
 	path = components/cam/src/physics/cosp2/external
 	url = git@github.com:CFMIP/COSPv2.0.git
 	branch = CESM_v2.1.4
-[submodule "cime"]
```

Comment:
It looks like submodule cime is missing completely. You also need to git submodule update following https://acme-climate.atlassian.net/wiki/spaces/ED/pages/1868455/Development+Getting+Started+Guide (maybe you already did, just mentioning in case). I don't think cime matters for what you're doing yet since it is so far only used when running with the old driver... but don't remove it's use for other folks!